monitoring/promfana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.0"
+version: 8.4.0
+appVersion: "Prometheus: v2.13.1; Grafana: v6.4.2"
```

Comment:
Should this be `"Prometheus: v9.2.0; Grafana: 4.0.1"` to match below?

Reply:
You are correct. I will update it tomorrow :D